README.md

```diff
@@ -8,7 +8,7 @@ If you have [Docker](https://www.docker.com/) installed, playing around with
 Carp is as easy as running:
 
 ```
-docker run -it carplang/carp
+docker run -it --rm carplang/carp
```

Comment:
--rm removes  container when it exits. (you can run `docker ps -a`) to see all containers. And you can remove them with `docker rm -f <container_id>`  or all containers with `docker rm -f $(docker ps -aq)`

Reply:
Yeah I guess it’s good practice to clean up after yourself :)